src/ui/print.jl

```diff
@@ -1,3 +1,13 @@
+function Base_type_depth_limit(io::IO; maxdepth=1)
+    sz = get(io, :displaysize, (typemax(Int), typemax(Int)))::Tuple{Int, Int}
```

Comment:
```suggestion
    sz = get(io, :displaysize, displaysize(io))::Tuple{Int, Int}
```
would be better?

Reply:
🤷🏻, I just copied what is in base to be honest. I'm fine with trying that, I'll replace it.